lib/cfn-nag/custom_rules/GameLiftFleetInboundPortRangeRule.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'base'
+
+class GameLiftFleetInboundPortRangeRule < BaseRule
+  def rule_text
+    'GameLift fleet EC2InboundPermissions found with port range instead of just a single port'
+  end
+
+  def rule_type
+    Violation::WARNING
+  end
+
+  def rule_id
+    'W65'
+  end
+
+  def audit_impl(cfn_model)
+    violating_gamelift_fleets = cfn_model.resources_by_type('AWS::GameLift::Fleet').select do |gamelift_fleet|
+      violating_permissions = gamelift_fleet.eC2InboundPermissions.select do |permission|
+        permission['FromPort'] != permission['ToPort']
```

Comment:
cfn sometimes allows for weak typing... i'm wondering that theoretically a developer could specify FromPort="666" and ToPort=666 and it'd be ok (so safer to convert both to string and then compare)

Reply:
Good call!  Fixed in the latest commit.